src/condense_solver.c

```diff
@@ -124,14 +146,18 @@ int condense_solver(int neq, double *x_f, double *abstol_f, double reltol_f,
                 return PMC_CONDENSE_SOLVER_SET_MAX_STEPS;
 
 
-        SUNLinearSolver LS = SUNLinSol_SPGMR(y, PREC_LEFT, 0);
-        if (condense_check_flag((void *)LS, "SUNLinSol_SPGMR", 0))
+#if SUNDIALS_VERSION_MAJOR >= 6
+	SUNLinearSolver LS = SUNLinSol_SPGMR(y, PREC_LEFT, 0, sunctx);
+#else
+	SUNLinearSolver LS = SUNLinSol_SPGMR(y, PREC_LEFT, 0);
+#endif
+	if (condense_check_flag((void *)LS, "SUNLinSol_SPGMR", 0))
                 return PMC_CONDENSE_SOLVER_LINSOL_CTOR;
 	flag = CVodeSetLinearSolver(cvode_mem, LS, NULL);
-        if (condense_check_flag(&flag, "CVodeSetLinearSolver", 1))
+	if (condense_check_flag(&flag, "CVodeSetLinearSolver", 1))
                 return PMC_CONDENSE_SOLVER_LINSOL_SET;
 	flag = CVodeSetPreconditioner(cvode_mem, NULL, condense_solver_Solve);
-        if (condense_check_flag(&flag, "CVodeSetPreconditioner", 1))
+	if (condense_check_flag(&flag, "CVodeSetPreconditioner", 1))
```

Comment:
What's happening with spaces versus tabs in this file. Ideally we should be using spaces only.

Reply:
I saw some inconsistency within the file and sought to remedy the inconsistency as I was modifying adjacent code. I agree it is using an odd convention. If you can let me know what the standard is, I am happy to do a whitespace-only formatting change if you'd like?

Reply:
We've been uses spaces (no tabs). But I'm hesitant to do a wholesale change because there are some large outstanding PRs that we should merge first to avoid formatting conflicts.

Reply:
I'm happy for this PR to be merged as is and we can worry about spaces/tabs later.